app/src/main/java/ar/com/wolox/android/example/ui/example/login/LoginPresenter.java

```diff
@@ -0,0 +1,23 @@
+package ar.com.wolox.android.example.ui.example.login;
+
+import javax.inject.Inject;
+
+import ar.com.wolox.wolmo.core.presenter.BasePresenter;
+
+/**
+ *
+ */
+class LoginPresenter extends BasePresenter<ILoginView> {
+
+    @Inject
+    LoginPresenter() {
+
+    }
+//    @Inject
```

Comment:
We can remove these commented lines

Reply:
✔ **Done** 

Also, LoginPresenter constructor is empty, so I updated it to the following:

```Java
@Inject
LoginPresenter() {}
```

In order to follow the convention indicated by the trainer